content/documentation/github.md

```diff
@@ -1,101 +1,143 @@
 
-## Installing an Existing Quire Project from GitHub
+### Save Changes
 
-1. Open GitHub Desktop and go to File > Clone Repository (Shift+Cmd+O).
+Open your project in your text editor (File > Open > [your project name]), make, and save changes. When you go into GitHub Desktop, you should see a list of all the files added, deleted, or edited in the left-hand pane. On the right, you'll see exactly what changes were made inside each file. Old text will be highlighted in red, while any new text will be highlighted in green. This enables you to compare and review changes easily.
 
-2. Click the URL tab, and enter the URL or GitHub username and repository name where indicated.
+{{< q-class "box tip" >}}
+- Save changes with either Command-S or File > Save. If working on multiple markdown files, make sure to save each file individually. You will see a little blue dot at the top of the page if there are any unsaved changes.   
+{{< /q-class >}}
 
-3. Click Clone.
+### Commit Changes
 
-Alternately, from the GitHub page of the project, click the green “Clone or download” button and select “Open in Desktop” and follow the prompts.
+At this point, your changes have been saved locally (i.e. only on your computer), but GitHub doesn’t know anything about them. For that, you need to make a [commit](https://docs.github.com/en/github/committing-changes-to-your-project/about-commits), or a small grouping of meaningful changes.
 
-You now have a copy of that project on your computer, that is still connected to the original copy on GitHub. If you make changes to it, they can be committed back to the original.
+To make a commit, write a required summary of the change you’re making (a longer description is optional) and click the "Commit to master" button. Keep in mind, a key advantage of GitHub is that you can undo your work by reverting commits and returning to earlier versions of your project. Here are a few tips for styling your commits to make that process easier:
 
-{{< q-class "box warning" >}}
+- Keep the commit name concise (around 50 characters max), so you can quickly and easily browse them in the future.
+- If you need to explain the commit in greater detail, use the description field (around 72 characters max).
+- The summary should be a phrase that completes the sentence “This commit will __”.
+- Use present/imperative tense, such as “Add contributor names and bios;” “Fix typos in section pages;” or “Change header color”.
+
+When naming commits, think of them as purposefully connected bundles that you could undo at a later date without effecting other areas of your project. Rather than committing all the random changes you made to a single file or everything you did before lunch that day, commit changes that *make sense together*.
 
-- In order to run and view the Quire preview of a project cloned from GitHub, you’ll need to first run `quire install` in your project directory.
+[[Include more tips / examples of good commits]]
```

Comment:
Not sure you really need to. Maybe there's a resource you can link to instead?

Reply:
Unfortunately, most resources for commit best practices are geared towards using `git commit` in the command-line which is just too confusing for newbies. But I think I covered the most important points.